content/en/docs/troubleshooting/ResourcePlacement.md

```diff
@@ -0,0 +1,206 @@
+---
+title: ResourcePlacement TSG
+description: Identify and fix KubeFleet issues associated with the ResourcePlacement API
+weight: 2
+---
+
+This TSG is meant to help you troubleshoot issues with the ResourcePlacement API in Fleet.
+
+## Resource Placement
+
+Internal Objects to keep in mind when troubleshooting RP related errors on the hub cluster:
+
+- `ResourceSnapshot`
+- `SchedulingPolicySnapshot`
+- `ResourceBinding`
+- `Work`
+
+Please read the [Fleet API reference](docs/api-reference) for more details about each object.
+
+## Complete Progress of the ResourcePlacement
+
+Understanding the progression and the status of the `ResourcePlacement` custom resource is crucial for diagnosing and identifying failures.
+You can view the status of the `ResourcePlacement` custom resource by using the following command:
+
+```bash
+kubectl describe resourceplacement <name> -n <namespace>
+```
+
+The complete progression of `ResourcePlacement` is as follows:
+
+1. `ResourcePlacementScheduled`: Indicates a resource has been scheduled for placement.
+   - If this condition is false, refer to [CRP Schedule Failure TSG](ClusterResourcePlacementScheduled).
+2. `ResourcePlacementRolloutStarted`: Indicates the rollout process has begun.
+   - If this condition is false, refer to [CRP Rollout Failure TSG](ClusterResourcePlacementRolloutStarted).
+   - If you are triggering a rollout with a staged update run, refer to [ClusterStagedUpdateRun TSG](ClusterStagedUpdateRun).
+3. `ResourcePlacementOverridden`: Indicates the resource has been overridden.
+   - If this condition is false, refer to [CRP Override Failure TSG](ClusterResourcePlacementOverridden).
+4. `ResourcePlacementWorkSynchronized`: Indicates the work objects have been synchronized.
+   - If this condition is false, refer to [CRP Work-Synchronization Failure TSG](ClusterResourcePlacementWorkSynchronized).
+5. `ResourcePlacementApplied`: Indicates the resource has been applied. This condition will only be populated if the
+apply strategy in use is of the type `ClientSideApply` (default) or `ServerSideApply`.
+   - If this condition is false, refer to [CRP Work-Application Failure TSG](ClusterResourcePlacementApplied).
+6. `ResourcePlacementAvailable`: Indicates the resource is available. This condition will only be populated if the
+apply strategy in use is of the type `ClientSideApply` (default) or `ServerSideApply`.
+   - If this condition is false, refer to [CRP Availability Failure TSG](ClusterResourcePlacementAvailable).
+7. `ResourcePlacementDiffreported`: Indicates whether diff reporting has completed on all resources. This condition
+will only be populated if the apply strategy in use is of the type `ReportDiff`.
+   - If this condition is false, refer to the [CRP Diff Reporting Failure TSG](ClusterResourcePlacementDiffReported) for more information.
+
+> **Note**: ResourcePlacement and ClusterResourcePlacement share the same underlying architecture and condition types.
```

Comment:
strictly speaking, they don't really share the same condition types (CRP vs RP) although they have 1 on 1 mapping